tests/FSharp.Test.Utilities/CompilerAssert.fs

```diff
@@ -337,7 +337,7 @@ type CompilerAssert private () =
         "tfm": "net5.0",
         "framework": {
             "name": "Microsoft.NETCore.App",
-            "version": "5.0.0"
+            "version": "6.0"
```

Comment:
This should probably stay at `5` and we should _also_ install the 5.0 runtime at CI time.